src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -84,6 +84,16 @@
 
 _ChipDeviceController_IterateDiscoveredCommissionableNodesFunct = CFUNCTYPE(None, c_char_p, c_size_t)
 
+# Defines for the transport payload types to use to select the suitable
+# underlying transport of the session.
+# class TransportPayloadCapability(ctypes.c_int):
+
+
+class TransportPayloadCapability(ctypes.c_int):
```

Comment:
Is this the direct equivalent of a C++ enum? If so please documenting that.
Other than that it would be good to use either "LARGE" or "TCP" consistently.
Also should this be a bit mask with one bit each for MRP and TCP?